OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
```

Comment:
Is this file required? We don't use Lighthouse anymore, I think it makes more sense to have this as a `.github/CODEOWNERS` file instead

Reply:
@swarnim-deepsource Can you take care of this?